moped-database/.gitignore

```diff
@@ -1,3 +1,4 @@
 atd_moped-full.sql.gz
 *moped_dump.sql
 production.sql
+reload.sh
```

Comment:
I'd like to include my reload script in here until we have time to enhance the `hasura-cluster` script with automatic loading from a snapshot functionality.

Reply:
rather than ignore it, any reason not to commit it? 😇

Reply:
```
#!/bin/bash

#docker exec -it -e PGPASSWORD=$MOPED_RR_PASSWORD moped-database-moped-pgsql-1 pg_dump -U $MOPED_RR_USERNAME -h moped-read-replica.austinmobility.io atd_moped > production.sql

./hasura-cluster stop
./hasura-cluster start_only
docker stop moped-database-hasura-1
echo 'DROP DATABASE moped; CREATE DATABASE moped' | docker exec -i moped-database-moped-pgsql-1 psql -U moped postgres;
docker exec -i moped-database-moped-pgsql-1 psql -U moped moped < production.sql; 
docker start moped-database-hasura-1
sleep 5
hasura migrate apply
hasura metadata apply
#echo 'UPDATE moped_users set user_id = 1 where user_id = 60;' | docker exec -i moped-database-moped-pgsql-1 psql -U moped moped;
```

I think we should probably spend time time integrating it properly into `hasura-cluster` since, at least in its current form, it relies on how I have my shell configured and what i export by default into my shell.

But if you think it'd be helpful, I'm totally game to drop it in there!! Just say the word!

Reply:
ok, that's fair! thanks for posting it here :)